frontend/src/components/tables/table-view.tsx

```diff
@@ -10,6 +10,8 @@ import { useListRows } from "@/lib/hooks"
 import { DataTable } from "@/components/data-table"
 import { TableViewAction } from "@/components/tables/table-view-action"
 import { TableViewColumnMenu } from "@/components/tables/table-view-column-menu"
+import { JsonViewWithControls } from "../workbench/events/events-selected-action"
+import { TooltipProvider } from "@radix-ui/react-tooltip"
```

Comment:
Prefer shadcn tooltip over the raw radix one
```suggestion
import { TooltipProvider } from "@/components/ui/tooltip"
```